addons/medical/stringtable.xml

```diff
@@ -3610,13 +3630,14 @@
         </Key>
         <Key ID="STR_ACE_Medical_BasicMedicalSettings_Module_Description">
             <English>Configure the treatment settings from ACE Basic Medical</English>
+            <German>Behandlungseinstellungen der Grundlegenden ACE-Medizin konfigurieren</German>
```

Comment:
Ist copy-paste vom darunterliegenden "advanced medical" Text der bereits vorhanden war. Ich habe das "grundlegende" mal in "einfache" umbenannt. Somit hätte man für Basic/Advanced im dt. einfach/erweitert. Aber so richtig griffig ist das auch noch nicht. Alternative Ideen?